src/ceph-volume/ceph_volume/tests/functional/lvm/centos7/bluestore/dmcrypt/group_vars/all

```diff
@@ -0,0 +1,30 @@
+---
+
+dmcrypt: True
+ceph_dev: True
+cluster: ceph
```

Comment:
Should we have at least one of these tests use a different cluster name?

Reply:
I don't know that would trigger anything different though. Or you mean for debugging?

Reply:
I was just wondering if it'd trigger some command that might have needed ``--cluster`` for any reason. If you don't think it's needed then no worries.

Reply:
dios mio man

I thought this was `cluster: test` in here, not `cluster: ceph`
![dios_mio_man](https://user-images.githubusercontent.com/317847/35299384-23f24350-0053-11e8-9bfa-1a86b3edbd49.gif)

Reply:
Custom cluster are no longer supported. 

http://lists.ceph.com/pipermail/ceph-users-ceph.com/2017-November/022244.html

Lets party